seed/views/00100_indivdual_capacity_view.js

```diff
@@ -7,10 +7,12 @@ exports.seed = function (knex, Promise) {
       , wpc.total_points
       , wpc.available_points
       , wpc.reduction_hours
-      , w.workload_owner_id AS id
+      , wo.id
+      , wo.contracted_hours
```

Comment:
This should probably be pulled from `wpc` where we put the default for the grade if there are no contracted_hours set..